framework/meta/src/cli_args/meta_build_args.rs

```diff
@@ -20,14 +19,24 @@ impl Default for BuildArgs {
             wat: false,
             extract_imports: true,
             target_dir: None,
-            abi_git_version: true,
         }
     }
 }
 
 impl BuildArgs {
-    pub fn process(args: &[String]) -> BuildArgs {
-        let mut result = BuildArgs::default();
+    pub fn default_dbg() -> Self {
```

Comment:
Consider renaming this to something else, since we already have a `default` implementation, and it could get confusing.

`new_debug_args()` maybe?